### ISSUE TITLE
Can't find xstate

### ISSUE THREAD

**Issue by vacquah** [sonntam/node-red-contrib-xstate-machine#41]
Hello -
I installed xstate from the node-red palette manager. ( v.1.2.2). Running node-red in an ubuntu 21.10 vm, node version 14.18.3, node-red version 2.1.6.
  
Everything shows up correctly, except in the editor, I get the error below. Any ideas?

![xstate](https://user-images.githubusercontent.com/2125909/151999145-c79cab2a-526f-4a66-b299-e1442dbb7750.jpg)

**Comment by sonntam** [comment]
Thank you for trying smxstate! 
Indeed, I could reproduce the problem when changeing the used code editor from "ace" to "monaco" in settings.js:
```js
editorTheme: {
    codeEditor: {
        lib: "monaco"
    }
}
```

With the "ace" editor no problems are shown. I'll have to take a look why.

**Comment by vacquah** [comment]
Not sure if this helps, but I have 2 installs of node-red. The first is on node-red version 2.1.6 ( where I noticed this problem ). That has the editor already set to "ace" but still has the problem.

I setup a fresh install - this time around, on version 2.2 of node-red, with "monaco" editor. Changing to "ace" on this one fixed the problem on it. 

Thank you for this module. Got a few questions about how to set somethings up. Email ?

**Comment by sonntam** [comment]
The problem has been fixed in V1.2.3.
